.github/actions/url-download-action/README.md

```diff
@@ -5,7 +5,7 @@
 
 # 🌍 Download Content from URL
 
-Uses WGET to download the content at a given URL.
+Uses WGET to downloads the content at a given URL.
```

Comment:
Why are you making `download` plural here? It's not good grammar. It should be

`Uses WGET to download the content from a given URL.`

Notice the change of `at` to `from` and the non-pluralization of `dowload`